src/home/CashInBottomSheet.tsx

```diff
@@ -110,9 +112,13 @@ function CashInBottomSheet() {
   const goToAddFunds = () => {
     onDismissBottomSheet()
 
-    navigate(Screens.FiatExchangeCurrency, {
-      flow: FiatExchangeFlow.CashIn,
-    })
+    getFeatureGate(StatsigFeatureGates.USE_CICO_CURRENCY_BOTTOM_SHEET)
```

Comment:
Given the number of places where we navigate to this screen, maybe we can have an helper function in NavigationService.ts that abstracts fetching the feature gate and navigating to appropriate screen?